js/stores/windowStore.js

```diff
@@ -742,6 +742,10 @@ const doAction = (action) => {
         windowState = windowState.setIn(path.concat(['security', 'isSecure']),
                                         action.securityState.secure)
       }
+      if (action.securityState.mixedContent !== undefined) {
```

Comment:
same thing as above. If this applies to both display and run then `insecureContent` would be ok, but if this is only for running then we should stick with `runInsecureContent`